### PR TITLE
changes to allow images to upload

### PR DESCRIPTION
**Pull request by ariannawilde3**
Here is my pull request so people can review it

### CODE REVIEW

**Review by jmdebarro — APPROVED** [review]
Looks great, we will likely have to delete some old outdated items from our db but nice change, 50mb is crazy

**Comment by Nikman800** [comment]
50mb? Dang, that's good.

File: packages/react-frontend/src/components/Post.js [hunk 1]
```diff
@@ -21,6 +21,20 @@ function Post() {
...
+      const file = event.target.files[0];
+      if (file) {
+        const reader = new FileReader();
+        reader.onload = (e) => {
+          setItem(prevItem => ({
+            ...prevItem,
+            image: e.target.result 
+          }));
+        };
+        reader.readAsDataURL(file);
```
Comment (Alecs-Games) [inline packages/react-frontend/src/components/Post.js:35]: This is a nice and straightforward way of doing it!

File: packages/express-backend/item-services.js [hunk 2]
```diff
@@ -29,12 +29,12 @@ async function connectToDB() {
...
-    name,
-    tags,
-    userId
-  );
+  // console.log(
+  //   "name: %s\ntags: %s, userId: %s\n\n",
+  //   name,
+  //   tags,
+  //   userId
+  // );
```
Comment (Alecs-Games) [inline packages/express-backend/item-services.js:37]: this console.log can probably be safely deleted

**Review by Alecs-Games — APPROVED** [review]
Looks good to me! Glad the file size was dealt with.

**Pull request merged**